sorc/link_workflow.sh

```diff
@@ -162,15 +162,14 @@ cd "${HOMEgfs}/parm/ufs" || exit 1
 ${LINK_OR_COPY} "${HOMEgfs}/sorc/ufs_model.fd/tests/parm/noahmptable.tbl" .
 
 cd "${HOMEgfs}/parm/post" || exit 1
-for file in postxconfig-NT-GEFS-F00.txt postxconfig-NT-GEFS.txt postxconfig-NT-GEFS-WAFS.txt \
-    postxconfig-NT-GEFS-F00-aerosol.txt postxconfig-NT-GEFS-aerosol.txt \
-    postxconfig-NT-GFS-ANL.txt postxconfig-NT-GFS-F00.txt postxconfig-NT-GFS-FLUX-F00.txt \
-    postxconfig-NT-GFS.txt postxconfig-NT-GFS-FLUX.txt postxconfig-NT-GFS-GOES.txt \
-    postxconfig-NT-GFS-F00-TWO.txt postxconfig-NT-GFS-TWO.txt \
-    params_grib2_tbl_new post_tag_gfs128 post_tag_gfs65 nam_micro_lookup.dat
```

Comment:
@DavidHuber-NOAA post_tag_gfs128 and post_tag_gfs65 should be linked to parm/post instead of parm/post/gfs.

Reply:
Alright, fixed.

Reply:
They are not used in the global-workflowl

Reply:
@aerorahul Could you let me know location of the 'itag' namelist for inline post?

Reply:
@WenMeng-NOAA @DavidHuber-NOAA 
I stand corrected.  We use the `post_tag_gfs128` file for inline post.
@WenMeng-NOAA This file should be in the `parm/gfs` directory like all other GFS parm files.
Will there be a `post_tag_gefs128` file for inline post for GEFS?

@WalterKolczynski-NOAA
What should we do? Maintain our copy or use the UPP provided `post_tag_gfsXXX`?

Reply:
@WenMeng-NOAA The global-workflow currently uses https://github.com/NOAA-EMC/UPP/blob/develop/parm/gfs/post_tag_gfs128 for inline post
It does not have `rdaod=.true.`

Reply:
@aerorahul @WalterKolczynski-NOAA Could these 'itag" namelists for gfs/gefs be saved in the global-workflow? It would be flexible.